packages/components/src/pages/ChatSearch.vue

```diff
@@ -423,9 +493,7 @@ export default {
         this.ask.on('stop', onStop);
 
         const explainRequest = {
-          question: message
-            .replace(/^@generate/, '@generate /format=xml')
-            .replace(/^@test/, '@test /format=xml'),
```

Comment:
Rebase issue here? I think `@test /format=xml` should be kept.

Reply:
I went back over my reflog and found where this change was introduced. It doesn't appear to have been part of a rebase, though the subject of the commit message is a little weird: it's labeled `!fixup !fixup`. I try to make a point of never fixing up a `!fixup` commit, so I'm not entirely sure how this happened. I think I missed it because it sort of looked like a formatting change, and I didn't notice that it was actually a behavior change.

In any event, this was the only unintentional change introduced in the `!fixup !fixup` commit. I also went back over all the changes in the PR, and didn't see anything else that looked out of place.

LMK if you have any other concerns I can address. Otherwise, I think this is ready to go.